Sources/FluentPostGIS/Support/EnablePostGISMigration.swift

```diff
@@ -1,14 +1,27 @@
+import FluentKit
 import Foundation
-import FluentPostgreSQL
+import SQLKit
 
 public struct EnablePostGISMigration: Migration {
-    public typealias Database = PostgreSQLDatabase
+    public init() {}
```

Comment:
Just double check the name of the migration is computed to be the same in Vapor 3 and Vapor 4 to make sure we don't try to run this again